README.md

```diff
@@ -25,26 +25,29 @@ See https://github.com/fossology/fossology/releases for source code download of
 
 For installation instructions see [Github Wiki](https://github.com/fossology/fossology/wiki)
 
-## Docker
-
-FOSSology comes with a Dockerfile allowing the containerized execution
-both as single instance or in combination with an external PostgreSQL database.
+### Install as a single container with all services combined
 **Note:** It is strongly recommended to use an external database for production
-use, since the the standalone image does not take care of data persistency.
+use, since the the single container does not handle data persistence.
+
+FOSSology comes with a [Dockerfile](https://github.com/fossology/fossology/blob/master/Dockerfile) which creates a single container which runs all three services; web-based UI, the FOSSology scheduler, and the PostgresSQL database. A pre-built Docker image is available from [Docker Hub](https://hub.docker.com/r/fossology/fossology/) and can be run using following commands:
 
-A pre-built Docker image is available from [Docker Hub](https://hub.docker.com/r/fossology/fossology/) and can be run using following command:
 ``` sh
 docker run -p 8081:80 fossology/fossology
 ```
+The docker image's web-base UI can then be used viewed http://IP_OF_DOCKER_HOST:8081/repo user fossy password fossy.
```

Comment:
I think that should be `http://IP_OF_DOCKER_HOST:80/repo` or `http://localhost:8081/repo` due to the `-p 8081:80`

Reply:
```suggestion
The docker image's web-base UI can then be used viewed http://IP_OF_DOCKER_HOST:8081/repo user `fossy` password `fossy`.
```